packages/dataregistry-extension/src/folders.tsx

```diff
@@ -34,10 +34,12 @@ function activate(
       async path =>
         new Set(
           [
-            ...(await fileBrowserFactory.defaultBrowser.model.manager.services.contents.get(
```

Comment:
Weird, must have been a new prettier version that changed this?

Reply:
Yeah, most likely. This is a drawback to relying on automatic code formatters, rather than just flagging lint errors. But c'est la vie.